### PR TITLE
Changes to update to TM tests setup play to use newly built  TM role

### PR DESCRIPTION
**Pull request by justjais**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/820

Changes to update to TM tests setup play to use newly built  TM role for making work the silent setup for TM over AWS instance.

### CODE REVIEW

**Comment by goneri** [comment]
recheck

File: zuul.d/project.yaml [hunk 1]
```diff
@@ -125,6 +125,7 @@
         - ansible-security-trendmicro-deepsec-appliance:
             files:
               - playbooks/ansible-network-appliance-base/.*
+              - roles/tm.*
```
Comment (goneri) [inline zuul.d/project.yaml:128]: You can remove the files section for your test. This way `ansible-security-tredmicro-deepsec-appliance` will be always executed, even if there is no changes in these directories

File: zuul.d/project.yaml [hunk 2]
```diff
@@ -122,9 +122,7 @@
               - playbooks/ansible-test-network-integration-base/files/bootstrap-splunk.yaml
               - playbooks/ansible-network-appliance-base/.*
               - roles/splunk.*
-        - ansible-security-trendmicro-deepsec-appliance:
-            files:
-              - playbooks/ansible-network-appliance-base/.*
+        - ansible-test-security-integration-trendmicro-deepsec-python38
```
Comment (goneri) [inline zuul.d/project.yaml:125]: You can also comment out the vmware and openvswitch jobs. Just to save some resources.

Comment (goneri) [reply zuul.d/project.yaml:125]: You did the change the `gate` section, you should actually add this line in the `check` part, e.g around line 85..

**Comment by justjais** [comment]
closing the PR as have transferred all of the content to PR #820

**Pull request closed without merging**